### DISCUSSION TITLE
parallel zfs offline backup while Sanoid / Syncoid active?

### DISCUSSION THREAD

**Discussion by spacerunner5** [jimsalterjrs/sanoid#787] [Q&A] [answered]
Hi,

first of all: Thank you for this awesome piece of software :) !

Assume I have 2 pools on 2 servers, using Sanoid / Syncoid for backup replication. Server B is pulling from Server A based on the Syncoid schedules / policies.

Question:

_Can I do an additional full backup (including all existing snaps) of the pool on Server A (to an external disk, creating a 1 disk pool, zfs send | recv to it) while Sanoid / Syncoid keeps running? What would be the way to go?_ 

Background: I can't stop Sanoid / Syncoid since that backup will take days. I do not care about speed or performance, but I'm a bit afraid of messing with Sanoid auto snaps etc. 

best & thanks for any hint,
stefan

**Comment by redmop** [comment, marked as answer]
Yes. I would suggest disabling syncoid's snapshots though (--no-sync-snap, I think). Just let it find the latest sanoid snapshot and use that to sync.

**Reply by spacerunner5** [reply]
Thank you 😄
